admin_guide/managing_networking.adoc

```diff
@@ -1364,6 +1364,8 @@ over multicast even when they can't communicate over unicast.)
 . Multicast packets sent by a pod in one project will never be delivered to pods
 in any other project, even if there are `NetworkPolicy` objects allowing
 communication between the to projects.
+endif::openshift-enterprise,openshift-origin[]
```

Comment:
@wgordon17 Looks like this is tripping up the build because it's missing its opening ifdef::openshift-enterprise,openshift-origin[]